src/_store/AppConfigProvider.js

```diff
@@ -23,9 +24,11 @@ export default class AppConfigProvider extends Component {
         language: PropTypes.string.isRequired,
         theme: PropTypes.string.isRequired,
     };
-
+    
     render() {
         const { children, language, theme } = this.props;
+        addLocaleData(localeData(locale(language)));
```

Comment:
You are right, there are other events i can explore since beside the ones that are called ones. Thanks for the correction.